src/frontend/citizen-portal/src/app/services/notice-of-dispute.service.ts

```diff
@@ -156,13 +154,11 @@ export class NoticeOfDisputeService {
   }
 
   createNoticeOfDispute(input: NoticeOfDispute): void {
-    input.issued_date = this.datePipe.transform(input.issued_date, "yyyy-MM-ddTHH:mm:ss");
```

Comment:
"YYYY-MM-ddTHH:mm:ssZ"

Reply:
results in a date that is not accepted by rest API